srcpkgs/inetutils/template

```diff
@@ -19,6 +19,11 @@ subpackages="inetutils-dnsdomainname inetutils-ftp inetutils-hostname
 
 CFLAGS="-fcommon"
 
+# broken tests
+checkdepends="iana-etc net-tools"
+# tests can't run in CI
+make_check=extended
```

Comment:
Even with the added check dependencies? :/

Reply:
even with the added check depedencias  the tests broke.
I will put it below the makedepends.